TrueCraft/Program.cs

```diff
@@ -54,20 +56,13 @@ public static void Main(string[] args)
 
         static void HandleChatMessageReceived(object sender, ChatMessageEventArgs e)
         {
-            // TODO: Make this more sophisticated
-            if (e.Message.StartsWith("/"))
+            if (e.Message[0] == '/')
             {
                 e.PreventDefault = true;
-                var Message = e.Message.Remove(0, 1);
-                var Command = Message.Trim();
-                var Arguments = new string[0];
-                if (Message.Split(' ').Length > 1)
-                {
-                    Command = Message.Split(' ')[0];
-                    Arguments = Message.Substring(Command.Length).Trim().Split(' ');
-                }
-
-                CommandManager.HandleCommand(e.Client, Command, Arguments);
+                var messageArray = e.Message.TrimStart('/') // replace with .Remove(0,1) if we're going to allow forward slashes in command aliases
+                                            .Split(new[] {' '},
+                                                StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Some indentation issues here. For these I prefer to indent it only one further from the previous line, instead of aligning it withe the previous line's method call.